addons/mail/static/src/chatter/web_portal/chatter.js

```diff
@@ -150,17 +122,16 @@ export class Chatter extends Component {
         this.state.isSearchOpen = false;
     }
 
-    async onClickAttachFile(ev) {
-        if (this.state.thread.id) {
-            return;
-        }
-        const saved = await this.props.saveRecord?.();
-        if (!saved) {
-            return false;
-        }
-    }
-
     onScroll() {
         this.state.isTopStickyPinned = this.rootRef.el.scrollTop !== 0;
     }
+
+    onUpdateProps(nextProps) {
```

Comment:
it is the same as onMount so maybe just make a single func to be called and give it a meaningful name

Reply:
Yes, as you mentioned in your other comment the behavior of these two methods are not same on the patch. I know it's strange to have them like this but I couldn't find a better way so far.

Reply:
it should be the same, see other comment. you can do it with providing two props and one of it can be undefined when in mount